frontend/assets/stylesheets/components/_hero-banner.scss

```diff
@@ -8,13 +8,18 @@
  */
 .hero-banner {
     color: $white;
+    height: 60vh;
     background-position: top center;
     background-repeat: repeat;
     background-attachment: fixed;
-    height: 60vh;
+    background-size: 100%;
+
+    @include mq(phablet) {
```

Comment:
Hah, yeah these names are a bit arbitrary. A phablet is a portmanteau of phone and tablet. ph-ablet.